src/main/java/com/github/platymemo/alaskanativecraft/mixin/client/InGameHudMixin.java

```diff
@@ -36,25 +37,26 @@
 
     @Inject(at = @At(value = "JUMP", opcode = Opcodes.IFEQ, ordinal = 1), method = "render")
     private void renderSnowGogglesOverlay(MatrixStack matrices, float tickDelta, CallbackInfo ci) {
-        if (this.client.options.getPerspective().isFirstPerson() && this.client.player.inventory.getArmorStack(3).getItem() == AlaskaItems.SNOW_GOGGLES) {
+        if (this.client.options.getPerspective().isFirstPerson() && this.client.player.getInventory().getArmorStack(3).getItem() == AlaskaItems.SNOW_GOGGLES) {
+            //TODO: test
```

Comment:
I see this TODO, has it been tested?

Reply:
Actually, no. I seem to have forgotten it, sorry.

However, 7df16722c2fd62282a0e91be0536d049e6fc80de implements a working fix